.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  examples-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #[macos-latest, ubuntu-latest]
+        python-version: [3.7]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install jina[cicd,test] --no-cache-dir
+      - name: Test examples with pytest
+        run: |
+          jina check
+          export JINA_LOG_VERBOSITY="ERROR"
+          pytest -s -v */tests
```

Comment:
southpark and object-search should not be tested in the same environment. We have to setup a fresh one per example. Otherwise the dependencies can collide.

Reply:
so one action per test? or is there a way to adapt it with a unique action?

Reply:
I guess you can parametrize this in the matrix above with the directory names. Not sure how, but that should be possible.

Reply:
Oh good idea, yes ! It is the way we test for `py3.7` , `py3.8`

Reply:
Let me check on this

Reply:
If we do this, then we have `pip install there` and avoid doing it from inside the `test` itself